test/Microsoft.Azure.WebJobs.Extensions.EventHubs.Tests/EventHubListenerTests.cs

```diff
@@ -222,5 +223,30 @@ public void GetMonitor_ReturnsExpectedValue()
 
             Assert.Same(scaleMonitor, scaleMonitor2);
         }
+
+        [Fact]
+        public void Dispose_Calls_StopAsync()
+        {                       
+            string functionId = "FunctionId";
+            string eventHubName = "EventHubName";
+            string consumerGroup = "ConsumerGroup";
+            var storageUri = new Uri("https://eventhubsteststorageaccount.blob.core.windows.net/");

```

Comment:
not used, hard coded on line 246 instead.  consider moving this to a class member as the same URI is used in GetMonitor_ReturnsExpectedValue() as well.